src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/csip/CSIPMappingResource.kt

```diff
@@ -267,6 +267,31 @@ class CSIPMappingResource(private val mappingService: CSIPMappingService) {
     csipId: String,
   ): CSIPReportMappingDto = mappingService.getMappingByDPSCSIPId(csipId)
 
+  @PostMapping("/nomis-csip-id")
```

Comment:
any reason this needs to a POST, could it not be a GET ? I'm guessing you are never excepting hundreds of CSIP Ids to be sent?